GoogleForCreators/web-stories-wp
# Templates: Add template colors to color picker

## Context
Noticed when adding template colors to the color presets that they weren't present in the color picker

## Summary
This PR adds a `Story colors` section of colors to the Color Picker:
<img width="637" alt="Screen Shot 2021-10-05 at 12 56 48 PM" src="https://user-images.githubusercontent.com/35983235/136077187-5f638e13-6ddc-4fab-a8d5-75b0296327a2.png">



## Relevant Technical Choices
NA


## To-do
NA


## User-facing changes
Now when a user opens up the color picker from this input:
<img width="1049" alt="Screen Shot 2021-10-05 at 12 57 44 PM" src="https://user-images.githubusercontent.com/35983235/136077369-b4f2ac67-e5c8-4f35-8b7d-c8e507f94b4c.png">

They have access to the Story Color presets populated from creating a story from a template.


## Testing Instructions





This PR can be tested by following these steps:

1. Create a story from a template
2. Click on a foreground element on the story
3. Go to the `Design`  -> `Style` panel (or `Design` -> `Text` for text) 
4. Click the circular color on this input to open the color picker
<img width="1049" alt="Screen Shot 2021-10-05 at 12 57 44 PM" src="https://user-images.githubusercontent.com/35983235/136077848-63c59c05-562e-4d46-ba24-1c2cac0de3f3.png">
5. See at the bottom of the color picker there's now a section called `Story Colors` that has all the colors from the template preview present:
<img width="1437" alt="Screen Shot 2021-10-05 at 1 03 15 PM" src="https://user-images.githubusercontent.com/35983235/136078154-399690cd-2870-46d9-b459-cf8bfbc99f75.png">





## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #9177
